kitsune/users/views.py

```diff
@@ -187,15 +187,51 @@ def deactivation_log(request):
     return render(request, "users/deactivation_log.html", {"deactivations": deactivations})
 
 
+def questions(request, username):
```

Comment:
similar to the comment in the `documents` view, this should become `questions_contributed`